hiedb.cabal

```diff
@@ -57,3 +57,28 @@ library
   hs-source-dirs:      src
   default-language:    Haskell2010
   ghc-options:         -Wall
+
+test-suite hiedb-tests
+    type:               exitcode-stdio-1.0
+    hs-source-dirs:     test
+    main-is:            Main.hs
+    build-tool-depends: hiedb:hiedb
```

Comment:
A trick I found on stack overflow - using this the test suite depends on hiedb binary built before tests are executed `cabal test`.